R/zyp.R

```diff
@@ -185,12 +185,17 @@ zyp.yuepilon <- function(y, x=1:length(y), conf.intervals=TRUE, preserve.range.f
   yt <- yt.prime[1:(n-1)] + trend * t.prime
   dmap.prime <- which(!is.na(yt))
   ytnm <- as.numeric(yt[dmap.prime])
+  t.prime <- t.prime[dmap.prime]
 
   # Calculate the Mann-Kendall test for significance on the blended series
-  Kend <- Kendall(t.prime[dmap.prime],ytnm)
+  Kend <- Kendall(t.prime,ytnm)
   tau <- Kend[1]
   Bsig <- Kend[2]
 
+  # Calculate slope on pre-whitened data

```

Comment:
@jameshiebert I went through Yue et al. 2002 paper "The influence of autocorrelation on the ability to detect trend in hydrological series." The zyp.yuepilon function in zyp.R follows the proposed procedure for detecting trends in significantly serially correlated hydrological series properly as written. However, the computation of the confidence intervals was not addressed directly in the paper in the same way it was for the zyp.zhang method. Therefore, I see how this was missed when the zyp.yuepilon method was coded up. The changes suggested to the computation of the confidence intervals and trend by @ateucher are correct in my opinion and his pull request should be accepted. Thanks for catching this @ateucher.